secp256k1-sys/build.rs

```diff
@@ -19,7 +19,7 @@ fn main() {
     base_config.include("depend/secp256k1/")
                .include("depend/secp256k1/include")
                .include("depend/secp256k1/src")
-               .flag_if_supported("-Wno-unused-function") // some ecmult stuff is defined but not used upstream
+               .flag("-Wno-unused-function") // some ecmult stuff is defined but not used upstream
```

Comment:
Looks like this needs bug report against `cc` (but we should check with the newest version first).

Reply:
Agreed. This change seems pretty sketchy.